SpatialGDK/Source/Private/Utils/ComponentFactory.cpp

```diff
@@ -286,6 +294,8 @@ TArray<Worker_ComponentData> ComponentFactory::CreateComponentDatas(UObject* Obj
 		ComponentDatas.Add(CreateHandoverComponentData(Info->SchemaComponents[SCHEMA_Handover], Object, Info, HandoverChangeState));
 	}
 
+	ComponentDatas.Add(CreateInterestComponentData(Object, Info));
```

Comment:
Don't we only support Actor interest currently?

Reply:
Yup, good catch.